Doc/whatsnew/3.14.rst

```diff
@@ -584,8 +587,24 @@ Improved error messages
                      ^^^^^^
    SyntaxError: cannot use subscript as import target
 
-.. seealso::
-   :pep:`649`.
+* Improved error message when trying to add an instance of an unhashable type
+  to a :class:`dict` or :class:`set`:
+
+  (Contributed by Victor Stinner in :gh:`426449d`.)
```

Comment:
This is the PR number, not the issue number, and I ignore the follow-up that fixed a typo in code